tools/image-remove-tag

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
```

Comment:
For this I don't think we need both the image sha1sum and tag.  The same tag cannot exist twice.  I may be missing something.

Other than needing to correctly redefine IMAGE_REPO,  image-delete is more in line with what untagging does unless there are multiple tags.   In theory it also loops over a list  of tags or sha1sums,  and works on either without defining both.

In any case,  we should probably just do one or the other,  but not a big deal which.

Reply:
Do you suggest updating `image-delete` to use IMAGE_REPO?  I'm fine with that, and ditching this script.

Reply:
Yeah,  whatever tweaks make it as easy as possible to use tags,  and just the tag:  (something)-latest or (something)-unscanned) or whatever.  Something simpler than $IMAGE_ID.